### ISSUE TITLE
Level Geometry errors 

### ISSUE THREAD

**Issue by HeadClot** [jbaicoianu/wadjs#3]
I have documented a few geometry rendering errors with WadJS. I have broken it down into three categories.

- Missing Geometry from the game world. Such as a wall or a floor. 
- Red, Blue, and Yellow Key Doors when opened display a geometry / texture where there should be nothing.
- Geometry is where it should not be. Such as a inset ceiling for example.

I am using Google Chrome version 69.0.3497.100 64 bit.

E1M1

https://i.imgur.com/ooFulph.png
https://i.imgur.com/MVXpAbU.jpg
https://i.imgur.com/weHevup.png

E1M2

https://i.imgur.com/G6RRx0P.png
https://i.imgur.com/bqa5tHz.png
https://i.imgur.com/sWVEvgZ.png
https://i.imgur.com/UtUzn3V.png

E1M3

https://i.imgur.com/bylL3qJ.png
https://i.imgur.com/ymHh1Xd.png
https://i.imgur.com/JxGNfhU.png

**Comment by jbaicoianu** [comment]
Yeah, I think we need to revisit our triangulation method to take sectors with holes into account
